SparkPost/2web2ui
# [FE-1329] Launch Hibana

FE-1329 - Launch Hibana

### What Changed

- Set the isHibanaEnabled value in HibanaContext to true
- Removed Theme Toggle from Profile page
- Removed cypress tests that passed for OG only
- Removed IS_HIBANA_ENABLED check
- Had to upgrade enzyme-adapter-react-16" to  "^1.15 since there were some randomly failing unit test
     - As a result had to upgrade snapshots



### How To Test
- its easier to review this PR with Hide white spaces turned on - https://github.com/SparkPost/2web2ui/pull/2015/files?diff=split&w=1
- Check if Hibana is the default and only theme of the app
- User should not be able to toggle back and forth between the OG and Hibana theme 

### To Do
